tools/webinstall/install.sh

```diff
@@ -1188,6 +1207,104 @@ install_kraftkit() {
     fi
 }
 
+# install_dependencies_gnu installs all kraftkit dependencies needed for
+# building and running unikernels on gnu distributions.
+# Returns:
+# Code: 0 on success, 1 on error
+install_dependencies_gnu() {
+    _idd_list=$(printf "%s%s"                       \
```

Comment:
psssst, Bash has _arrays_ 🤫

Reply:
we are using `sh` here, no fancy bash

Reply:
fancy new picture :sweat_smile:

Reply:
> we are using `sh` here, no fancy bash

We really have zero self respect 😢

Reply:
you are free to add the `ps1` version, no need to also do `bat` :wink: